sqlite.go

```diff
@@ -1128,6 +1128,64 @@ func (stmt *Stmt) GetLen(colName string) int {
 	return stmt.ColumnLen(col)
 }
 
+// Limit is a category of performance limits.
```

Comment:
Wherever possible in the documentation I want to provide convenient relevant links to the SQLite docs. 

https://sqlite.org/c3ref/c_limit_attached.html